nvim/init.lua

```diff
@@ -370,6 +555,23 @@ require('lazy').setup({
     config = function(_, opts) require 'lsp_signature'.setup(opts) end
   },
 
+  {
+    "rockerBOO/symbols-outline.nvim",
```

Comment:
This plugin has been archived. Also LspSaga has same functionality (code outline)

Reply:
Him well spotted this plugin has very powerful feature for renaming similar to Java plugin when nvtree Do you know can we have a similar functionality from LspSaga

Reply:
Can you elaborate a bit more here on what functionality this plugin has that LspSaga doesn't?

Reply:
Actually we are using a fork which maintained from very famoust plugin developer https://github.com/rockerBOO/symbols-outline.nvim
I'm talking about rename symbols option which is very powerful and could be found when LspSaga. Try it and let me know what you think

https://github.com/rockerBOO/symbols-outline.nvim?tab=readme-ov-file#default-keymaps